### ISSUE TITLE
Disable extension browser monitoring for specific websites

### ISSUE THREAD

**Issue by ModiGrey** [subhra74/xdm#1121]
**Describe the bug**
Some websites cause issues with the browser monitoring extension, and adding a website to the "do not automatically capture downloads from below websites" does not seem to resolve the issue.

**To Reproduce**
1. Go to 'makeuseof.com' and multiple annoying pop-ups of captured JPGs will stack upon each other.
2. or go to 'twitch.tv'
3. open any live stream
4. see the number on extension icon increases indefinitely.

**Expected behavior**
An option for the user to be able to disable extension browser monitoring for specific websites.

**please complete the following information:**
 - OS: Windows
 - Browser Firefox, and Chrome
 - XDM addon Version [v8+]
 - XDM Version [8.0.29 BETA]

**Comment by inuonly** [comment]
I have same issue. It generatd literally 1000+ and keeps going till 2000+ and more like never stops increasing links in some time. 
![image](https://github.com/user-attachments/assets/089faf7d-9474-4284-863c-55dd66e830f9)

This happens on firefox as well. After clearing within some time it started increasing the counter again. Sometimes it does not generate the link. 
![image](https://github.com/user-attachments/assets/f808bcda-b747-4291-828d-9276735c783b)
If I try to download it, it is usually a picture card from my home assistant. 
Sometimes it is my camera feed from some other weebsite like motion eye etc etc . In short any updating image card or video is captured ( I suppose ) and keeps counter increasing based on frames updated on the live feed of cameras being viewed in browser anytime (thou not sure this is my behaviour guess based on my examples) . 

TLDR : So yeah I have same issue. Screenshots attached for sample.
